eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -422,14 +422,6 @@ jobs:
         publishRunAttachments: true
         testRunTitle: SourceBuild_SmokeTests_$(Agent.JobName)
 
-  - publish: '$(sourcesPath)/artifacts/assets/Release/'
+  - publish: '$(sourcesPath)/artifacts'
```

Comment:
We don't want to upload the entire artifacts folder. Only artifacts/packages, artifacts/assets and the artifacts/MergedManifest.xml (or whatever the name is for that).

Reply:
yup, I saw there was a bunch of other things in there that we didn't need. Fixed